tests/lib/test_pkg.ml

```diff
@@ -95,22 +95,33 @@ let distrib_file =
       "_build/foo-v0.tbz";
   ]
 
-let distrib_filename =
-  let make_test ~test_name ~opam =
-    let test_name = "distrib_filename: " ^ test_name in
-    make_test ~test_name ~name:"foo" (Pkg.distrib_filename ~opam)
+let distrib_opam_path =
+  let make_test ~test_name =
+    let test_name = "distrib_opam_path: " ^ test_name in
+    make_test ~test_name ~name:"foo" Pkg.distrib_opam_path
   in
   [
-    make_test ~test_name:"1" ~opam:false ~tag:"v0" "foo-v0";
-    make_test ~test_name:"2" ~opam:true ~tag:"v0" "foo.0";
-    make_test ~test_name:"3" ~opam:false ~version:"v0" "foo-v0";
-    make_test ~test_name:"4" ~opam:true ~version:"v0" "foo.v0";
-    make_test ~test_name:"5" ~opam:false ~tag:"v0" ~keep_v:false "foo-v0";
-    make_test ~test_name:"6" ~opam:true ~tag:"v0" ~keep_v:false "foo.0";
-    make_test ~test_name:"7" ~opam:false ~tag:"v0" ~keep_v:true "foo-v0";
-    make_test ~test_name:"8" ~opam:true ~tag:"v0" ~keep_v:true "foo.v0";
-    make_test ~test_name:"9" ~opam:false ~tag:"v0" ~version:"x" "foo-v0";
-    make_test ~test_name:"10" ~opam:true ~tag:"v0" ~version:"x" "foo.x";
+    make_test ~test_name:"from version while keeping v" ~version:"v0"
+      ~keep_v:true "foo.v0";
```

Comment:
Do this tests still make sense since the version is used directly and therefore keep_v has no effect?

Wouldn't it be better to test how `Pkg.version` and `Pkg.tag` work instead, pontentially in a separate PR?

Reply:
Honestly, I don't think they do. I had to remove some already, because there's no way to unit test these without running them in an existing git repo (so I converted them into an integration test to still keep the functionality tested) and the remaining ones are a bit nonsensical because they mostly test absolutely trivial things while looking like the expected cases are wrong (`~keep_v:false` yet the expected value has a `v` — I'm thinking that `keep_v` should be also renamed/refactored at a later point to something like `type version_transform = [`As_is, `Remove_v_from_tag]` but that's probably something for a future PR).

So I guess I'll just delete them and make a PR in the future to test this better.